src/keyboard.js

```diff
@@ -43,7 +43,7 @@ export const keyCodes = generateKnownKeyCodes();
  *
  * Note: Key names are matched with {@link module:utils/keyboard~keyCodes} in a case-insensitive way.
  *
- * @param {String|module:utils/keyboard~KeystrokeInfo} Key name (see {@link module:utils/keyboard~keyCodes})
+ * @param {String|module:utils/keyboard~KeystrokeInfo} key name (see {@link module:utils/keyboard~keyCodes})
```

Comment:
That's a sentence so it should start with an uppercased letter.

Reply:
When looking in documentation (https://ckeditor.com/docs/ckeditor5/latest/api/module_utils_keyboard.html#static-function-getCode), there is

```
getCode( Key ) → Number
```

Although all functions always have they parameters with lower case. Also in the _Parameters_ part, sentence is broken between "key" and "name".

What about the following code instead (twice key)?

```
* @param {String|module:utils/keyboard~KeystrokeInfo} key Key name (see {@link module:utils/keyboard~keyCodes})
```

By the way, how do you generate the documentation? I did not find anything in the doc about it. Do you use internal tools?

Reply:
Hi, @denisname!

> By the way, how do you generate the documentation? I did not find anything in the doc about it. Do you use internal tools?

Yes, we use the `JSDoc3` package with a few custom plugins available here - https://github.com/ckeditor/ckeditor5-dev/tree/master/packages/jsdoc-plugins to generate proper AST tree and then we generate the documentation using our internal tool called [`Umberto`](https://www.npmjs.com/package/umberto).

> What about the following code instead (twice key)?

> `* @param {String|module:utils/keyboard~KeystrokeInfo} key Key name (see {@link module:utils/keyboard~keyCodes})`

Yep, you're right, the sentence should include both `keys` as the first key must match the param's name.

Btw I've proposed https://github.com/ckeditor/ckeditor5/issues/1415, it should solve many of such typos and small documentation errors.